scripts/make/qemu.mk

```diff
@@ -24,7 +24,17 @@ qemu_args-aarch64 := \
   -machine virt \
   -kernel $(OUT_BIN)
 
+LOONGARCH_BIOS = tools/la64/loongarch_bios_0310.bin
+qemu_args-loongarch64 := \
+  -bios $(LOONGARCH_BIOS) \
```

Comment:
It seems that it can be boot upon default bios provided by QEMU 8.1.94,
I'll try to verify other applications later, why not just delete this line?
![屏幕截图 2023-12-26 141649](https://github.com/rcore-os/arceos/assets/50011172/a06300c6-a8bd-486a-a159-5b65a8f67e79)

Reply:
There should be no bios here.

![image](https://github.com/rcore-os/arceos/assets/57819926/cd8327e3-5767-4e4f-851e-bbd535d7a555)

The first instruction is not located in the BIOS.

Reply:
The latest version of la qemu seems to have changed the memory layout, which is causing some loading errors. Also, We need to use the environment prepared by the BIOS for us, such as multi-core startup.
![image](https://github.com/rcore-os/arceos/assets/57819926/16e5edf2-520f-472e-a5ff-5e0188ca1f48)

Reply:
I see, I tried both SMP=2 and SMP=4 and they got stucked at different place.
Sorry I haven't try your provided QEMU emulator, I appreciate and admire your excellent work.
But @equation314 suggested that code should run on mainstream QEMU version (by the way I personally do not recommend that the system itself should rely a special BIOS version as loongaarch currently provides), maybe we should have a little discussion about it?

Reply:
Is it possible to use mainline QEMU with the specific BIOS?

Reply:
I will try it.

Reply:
Now, we can use the mainline QEMU for la64. 
I discussed the issue of mainline  qemu with some teachers at loongarch. They said that mainline qemu cannot currently boot the ELF format kernel. because our kernel is not as complex as Linux, we do not need the information passed by the bios, so here I chose to remove the BIOS dependency and start the kernel directly.

Reply:
I tested it on qemu-8.2.0. The latest rust tool chain seems to cause several programs under apps/c/pthread to fail.

Reply:
provide a list of the current support status of apps?

Reply:
These apps have not been tested: apps/c/iperf|redis|sqlite3. Other apps have been tested.